Tasks/VsTestV3/typings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
What does this file do?
Do we need it?

Reply:
Added this file for sync between both versions.